contracts/test/MockHyperdriveDeployer.sol

```diff
@@ -8,6 +8,10 @@ import { IHyperdriveTargetDeployer } from "contracts/src/interfaces/IHyperdriveT
 import { MockHyperdrive } from "./MockHyperdrive.sol";
 
 contract MockHyperdriveDeployer is IHyperdriveDeployerCoordinator {
+    string public constant name = "MockHyperdriveDeployer";
+
+    string public constant version = "v1.0.0";
```

Comment:
wrong version. should use constant, right?

Reply:
Good catch!